src/map/unit.cpp

```diff
@@ -1688,9 +1694,14 @@ bool unit_stop_walking( block_list* bl, int32 type, t_tick canmove_delay ){
 		return false;
 	}
 
-	ud = unit_bl2ud(bl);
+	if (ud = unit_bl2ud(bl); ud == nullptr)
```

Comment:
Please dont put the assignment into the if, unless you declare the variable in this scope.

Reply:
Ok, I thought that is the preferred style. But no problem, can change.

Reply:
Fixed.